components/notification/overview.md

```diff
@@ -66,147 +56,79 @@ To add a Telerik Notification component to your page:
+@* The fully qualified class name of the notification component so you can use its reference *@
 
-<TelerikButton OnClick="@AddNotifications">Add success, info, warning, error notification</TelerikButton>
+<TelerikButton OnClick="@AutoCloseNotification">Add auto close notification</TelerikButton>
 
-<TelerikNotification @ref="@NotificationReference" Class="MyTelerikNotification"></TelerikNotification>
+<TelerikNotification @ref="@NotificationReference"></TelerikNotification>
 
 @code {
-    public TelerikNotification NotificationReference { get; set; }
+    Telerik.Blazor.Components.TelerikNotification NotificationReference { get; set; }
 
-    public void AddNotifications()
+    void AutoCloseNotification()
     {
-        // Success
-        NotificationReference.Show(new NotificationModel()
-        {
-            ThemeColor = ThemeConstants.Notification.ThemeColor.Success,
-            Text = "Success",
-        });
-
-        // Info
         NotificationReference.Show(new NotificationModel()
         {
-            ThemeColor = ThemeConstants.Notification.ThemeColor.Info,
-            Text = "Info",
-        });
-
-        // Warning
-        NotificationReference.Show(new NotificationModel()
-        {
-            ThemeColor = ThemeConstants.Notification.ThemeColor.Warning,
-            Text = "Warning",
-        });
-
-        // Error
-        NotificationReference.Show(new NotificationModel()
-        {
-            ThemeColor = ThemeConstants.Notification.ThemeColor.Error,
-            Text = "Error",
+            Text = "Auto Closable Notification",
+            ThemeColor = "primary",
+            Closable = false
         });
     }
-}
-
-<style>
-    .MyTelerikNotification .k-notification-container .k-notification-wrap {
-        width: 300px;
-        height: 50px;
-        font-size: 1.5em;
-        text-align: center;
-        align-items: center;
-    }
-</style>
+} 
 ````
 
+## Stacked Notifications
 
+Multiple Notifications can stack if they derive from different references. [Read the Stacked Notifications article for more information...]({%slug notification-stacked-notifications%})
 
-## Features
-
->caption The Notification provides the following features:
-
-* `Class` - `string` - the CSS class that will be rendered on the main wrapping element of the Notification component. You could use that class to to control things like the size or the z-index of the component. You can find more infomation and examples in the [Appearance]({%slug notification-appearance%}) article.
+## Styling and Appearance
 
-* `AnimationType` - an `enum` which allows you to customize the animation of the Notifications. You can find more infomation and examples in the [Appearance]({%slug notification-appearance%}) article.
+You can customize the styling and the appearance of the Notification. [Read the Appearance article for more information...]({%slug notification-appearance%})
```

Comment:
Feature sections serve 3 purposes - 
* hint about component capabilities
* improve SEO
* direct to other pages for more information

Brief one-sentence sections may not bring enough value to the reader and can make the documentation look "auto generated". It takes very little to improve all 3 roles by using relevant key words in the link and mention some technical aspects of the feature.

Also, please add a section for the Templates.

```suggestion
You can [customize the styling and the appearance of the Notification]({%slug notification-appearance%}). For example, select the animation type or choose a suitable background color.
```